engine-helpers/src/index.ts

```diff
@@ -1021,4 +1041,34 @@ export class WWTInstance {
         options.format);
     });
   }
+
+  /** Capture a video as a sequence of frames using the given parameters
+   *
+   * This function returns a readable stream whose values are the exported frames.
+  */
+  captureVideo(options: CaptureVideoOptions): ReadableStream<Blob | null> {
```

Comment:
Does the type yielded here need to be `Blob | null` or can it just be `Blob`? Does the stream yield `null` once it's closed, or something?

Reply:
It's `Blob | null` to match `BlobCallback` - the argument to the callback in `HTMLCanvasElement::toBlob`, which is where the blob-relevant code ultimately gets executed, is `null` if the image can't be created for some reason.